bin/metrics/cli.js

```diff
@@ -0,0 +1,6 @@
+const {main} = require('./main');
```

Comment:
Please use shebang `#!/usr/bin/env node` in the first line

Reply:
👍